test/template_handler_test.rb

```diff
@@ -31,25 +31,25 @@ def source
 
   test 'typescript views are served as javascript' do
     get '/site/index.js'
-    assert_match /var x = 5;\s*/,
+    assert_match (/var x = 5;\s*/),
```

Comment:
What effect do the brackets have? I didn't see any difference after removing them when running tests.

Reply:
I removed warnings.

```shell
❯ bundle exec rake test --trace             
** Invoke test (first_time)
** Execute test
/Users/mason/workspace/NEWROPE/typescript-rails/test/template_handler_test.rb:34: warning: ambiguous first argument; put parentheses or a space even after `/' operator
/Users/mason/workspace/NEWROPE/typescript-rails/test/template_handler_test.rb:40: warning: ambiguous first argument; put parentheses or a space even after `/' operator
/Users/mason/workspace/NEWROPE/typescript-rails/test/template_handler_test.rb:46: warning: ambiguous first argument; put parentheses or a space even after `/' operator
/Users/mason/workspace/NEWROPE/typescript-rails/test/template_handler_test.rb:52: warning: ambiguous first argument; put parentheses or a space even after `/' operator
Run options: --seed 34434
```